packages/manager/modules/pci/src/projects/new/payment/components/challenge/index.scss

```diff
@@ -0,0 +1,36 @@
+pci-project-new-payment-challenge {
+  /* stylelint-disable-line */
```

Comment:
Can't we go without disabling the linter here ?

Reply:
you're right. This is not needed anymore